internal/layer/windows_writer.go

```diff
@@ -0,0 +1,95 @@
+package layer
+
+import (
+	"archive/tar"
+	"io"
+	"path"
+	"strings"
+)
+
+type WindowsWriter struct {
```

Comment:
I'm thinking this whole package should move into `imgutil`, as it's going to be needed by `lifecycle` for exporting images, etc.

Reply:
Moved out of this repo. Will show up later in `imgutil`.